sites/public/pages/account/favorites.tsx

```diff
@@ -55,54 +55,50 @@ const FavoritedListingsPage = () => {
     OrderByFieldsEnum.mostRecentlyUpdated
   )
 
-  const pageTitle = `${t("pageTitle.favorites")} - ${t("nav.siteTitle")}`
-  const metaDescription = t("pageDescription.welcome", { regionName: t("region.name") })
-  const metaImage = "" // TODO: replace with hero image
+  const content = useMemo(() => {
+    if (!profile || listingsLoading) {
+      return null
+    } else if (!filterState.favorited) {
+      return (
+        <div className="p-8">
+          <h2 className="pb-4">{t("account.noFavorites")}</h2>
+          <LinkButton href="/listings">{t("listings.browseListings")}</LinkButton>
+        </div>
+      )
+    }
+    return (
+      <>
+        {!listingsError && listingsData?.meta.totalItems === 0 && (
```

Comment:
I'm curious why we have this no results state at all. If there are no results, shouldn't we just show the `no favorites` state? I can't get this state to trigger.